### PR TITLE
Add mining beneficiary and block reward.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Describe the change

Add the genesis file parameters to add a mining beneficiary and block reward.

### CODE REVIEW

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 1]
```diff
@@ -60,10 +62,15 @@ The properties specific to IBFT 2.0 are:
 * `blockperiodseconds` - The minimum block time, in seconds.
 * `epochlength` - The number of blocks after which to reset all votes.
 * `requesttimeoutseconds` - The timeout for each consensus round before a round change, in seconds.
+* `blockreward` - Optional reward amount in Wei to reward the beneficiary. Defaults to zero (0).
+    Can be specified as a hexidecimal or decimal string value.
```
Comment (rain-on) [inline docs/HowTo/Configure/Consensus-Protocols/IBFT.md:66]: is it worth saying hexidecimal (with 0x prefix) or decimal string value?

Comment (bgravenorst) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md]: Done

File: docs/HowTo/Configure/Consensus-Protocols/IBFT.md [hunk 2]
```diff
@@ -60,10 +62,15 @@ The properties specific to IBFT 2.0 are:
 * `blockperiodseconds` - The minimum block time, in seconds.
 * `epochlength` - The number of blocks after which to reset all votes.
 * `requesttimeoutseconds` - The timeout for each consensus round before a round change, in seconds.
+* `blockreward` - Optional reward amount in Wei to reward the beneficiary. Defaults to zero (0).
+    Can be specified as a hexidecimal (with 0x prefix) or decimal string value.
+* `miningbeneficiary` - Optional beneficiary of the `blockreward`. Defaults to the validator
```
Comment (MadelineMurray) [inline docs/HowTo/Configure/Consensus-Protocols/IBFT.md:67]: @rain-on - are `blockreward` and `miningbeneficiary` network level settings or node level settings?

Comment (rain-on) [reply docs/HowTo/Configure/Consensus-Protocols/IBFT.md:67]: They MUST be network level - if someone in the network has a value different to that of the block proposer, the block import will fail on that node (due to a state-root mismatch, i.e. proposer paid blockReward to address-X, receiving node paid address-Y, meaning the world state at the end of the block is different between the nodes).

**Pull request merged**